enterprise_access/apps/content_assignments/management/commands/tests/test_automatically_nudge_assignments.py

```diff
@@ -0,0 +1,587 @@
+                    'enroll_by_date': enrollment_end.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'content_price': 123,
+                },
+                'course_type': 'executive-education-2u',
+            },
+            'edX+edXPrivacy101': {
+                'normalized_metadata': {
+                    'start_date': start_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'end_date': end_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'enroll_by_date': enrollment_end.strftime("%Y-%m-%d %H:%M"),
+                    'content_price': 321,
+                },
+                'course_type': 'executive-education',
+            },
+            'edX+edXTesseract4D': {
+                'key': 'edX+edXTesseract4D',
+                'normalized_metadata': {
+                    'start_date': start_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'end_date': end_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'enroll_by_date': enrollment_end.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'content_price': 123,
+                },
+                'course_type': 'executive-education-2u',
+            },
+            'edX+edXQuadrilateral306090': {
+                'key': 'edX+edXQuadrilateral306090',
+                'normalized_metadata': {
+                    'start_date': start_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'end_date': end_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'enroll_by_date': enrollment_end.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'content_price': 123,
+                },
+                'course_type': 'executive-education-2u',
+            },
+            'edX+edXIsoscelesPyramid2012': {
+                'key': 'edX+edXIsoscelesPyramid2012',
+                'normalized_metadata': {
+                    'start_date': start_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'end_date': end_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'enroll_by_date': enrollment_end.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'content_price': 123,
+                },
+                'course_type': 'executive-education-2u',
+            },
+            'edX+edXBeeHivesAlive0220': {
+                'key': 'edX+edXBeeHivesAlive0220',
+                'normalized_metadata': {
+                    'start_date': start_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'end_date': end_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'enroll_by_date': enrollment_end.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    'content_price': 123,
+                },
+                'course_type': 'executive-education-2u',
+            },
+        }
+
+        call_command(self.command, days_before_course_start_date=14)
+
+        mock_send_reminder_email_for_pending_assignment_task.assert_has_calls([
+            call(self.alice_assignment.uuid),
+            call(self.bob_assignment.uuid),
```

Comment:
This assertion will change when you change the filter from accepted to allocated.